src/gnssro/gnssro_bufr2ioda.f90

```diff
@@ -326,96 +330,151 @@ program gnssro_bufr2ioda
-call check( nf90_put_att(ncid, varid_bnd, "longname", "Bending Angle" ))
-call check( nf90_put_att(ncid, varid_bnd, "units", "Radians" ))
-call check( nf90_put_att(ncid, varid_bnd, "valid_range", "-0.001 - 0.08 Radians" ))
-call check( nf90_def_var(ncid, "bending_angle@ObsError", NF90_FLOAT, nlocs_dimid, varid_bndoe) )
-call check( nf90_put_att(ncid, varid_bndoe, "longname", "Input error in Bending Angle" ))
-call check( nf90_put_att(ncid, varid_bndoe, "units", "Radians" ))
-call check( nf90_put_att(ncid, varid_bndoe, "valid_range", "0 - 0.008 Radians" ))
-call check( nf90_def_var(ncid, "impact_parameter@MetaData", NF90_FLOAT, nlocs_dimid, varid_impp))
-call check( nf90_put_att(ncid, varid_impp, "longname", "distance from centre of curvature" ))
-call check( nf90_put_att(ncid, varid_impp, "units", "Meters" ))
-call check( nf90_put_att(ncid, varid_impp, "valid_range", "6200 - 6600 km" ))
-call check( nf90_def_var(ncid, "impact_height@MetaData", NF90_FLOAT, nlocs_dimid, varid_imph))
-call check( nf90_put_att(ncid, varid_imph, "longname", "distance from mean sea level" ))
-call check( nf90_put_att(ncid, varid_imph, "units", "Meters" ))
-call check( nf90_put_att(ncid, varid_imph, "valid_range", "0 - 200 km" ))
-call check( nf90_def_var(ncid, "sensor_azimuth_angle@MetaData", NF90_FLOAT, nlocs_dimid, varid_azim))
-call check( nf90_put_att(ncid, varid_azim, "longname", "GNSS->LEO line of sight" ))
-call check( nf90_put_att(ncid, varid_azim, "_FillValue", real(missingvalue) ))
-call check( nf90_put_att(ncid, varid_azim, "units", "Degree" ))
-call check( nf90_put_att(ncid, varid_azim, "valid_range", "0 - 360 degree" ))
-call check( nf90_def_var(ncid, "geoid_height_above_reference_ellipsoid@MetaData",NF90_FLOAT, nlocs_dimid, varid_geoid))
-call check( nf90_put_att(ncid, varid_geoid, "longname", "Geoid height above WGS-84 ellipsoid" ))
-call check( nf90_put_att(ncid, varid_geoid, "units", "Meters" ))
-call check( nf90_put_att(ncid, varid_geoid, "valid_range", "-200 - 200 m" ))
-call check( nf90_def_var(ncid, "earth_radius_of_curvature@MetaData",NF90_FLOAT, nlocs_dimid, varid_rfict))
-call check( nf90_put_att(ncid, varid_rfict, "longname", "Earth’s local radius of curvature" ))
-call check( nf90_put_att(ncid, varid_rfict, "units", "Meters" ))
-call check( nf90_put_att(ncid, varid_rfict, "valid_range", "6200 - 6600 km" ))
+call check( nf90_put_att(ncid, NF90_GLOBAL, 'date_time', anatime) )
+call check( nf90_put_att(ncid, NF90_GLOBAL, 'ioda_version', 'fortran generated ioda2 file') )
+call check( nf90_def_grp(ncid, 'MetaData', grpid_metadata) )
+call check( nf90_def_grp(ncid, 'ObsValue', grpid_obsvalue) )
+call check( nf90_def_grp(ncid, 'ObsError', grpid_obserror) )
+
+call check( nf90_def_var(grpid_metadata, "latitude",      NF90_FLOAT, nlocs_dimid, varid_lat) )
+call check( nf90_put_att(grpid_metadata, varid_lat, "_FillValue",real(r_missing) ))
+call check( nf90_put_att(grpid_metadata, varid_lat, "units",  "degree" ))
```

Comment:
Please change units to "degrees_north" to match JEDI conventions. Thanks!

Reply:
done!